audbackend/core/interface/versioned.py

```diff
@@ -59,6 +60,62 @@ def checksum(
         path_with_version = self._path_with_version(path, version)
         return self.backend.checksum(path_with_version)
 
+    def copy_file(
+            self,
+            src_path: str,
+            dst_path: str,
+            *,
+            version: str = None,
+            verbose: bool = False,
+    ):
+        r"""Copy file on backend.
+
+        If ``version`` is ``None``
+        all versions of ``src_path``
+        will be copied.
+
+        If ``dst_path`` exists
+        with a different checksum,
+        it is overwritten,
+        or otherwise,
+        the operation is silently skipped.
```

Comment:
As above:

```suggestion
        If ``dst_path`` exists
        and has a different checksum,
        it is overwritten.
        Otherwise,
        the operation is silently skipped.
```